stories/darnah-flood.stories.mdx

```diff
@@ -0,0 +1,305 @@
+  </Prose>
+</Block>
+
+
+<Block>
+  <Prose>
+    ## Medicane Daniel
+    [Medicanes](https://www.google.com/url?q=https://www.easa.europa.eu/community/topics/medicanes&sa=D&source=docs&ust=1709231595506704&usg=AOvVaw3IE4-Z1b9q1j7MKe7dE4YH) have similar properties to hurricanes, as they are warm-core and quasi-tropical in nature, but are not technically designated as hurricanes. Storm Daniel formed in the Ionian Sea on September 4th, 2023, and drifted to the south underneath an Omega Block (a high pressure system stuck in place between two low-pressure systems). As the cyclone moved out of the Ionian Sea and into the central Mediterranean Sea on September 5th, it transitioned into a medicane. Daniel began to drift southeastward towards the northern Libyan coast on September 9th before making landfall near the city of Benghazi on September 10th. Satellite-derived measurements from the ASCAT (Advanced Scatterometer) recorded wind speeds of up to 85 km/hour before Daniel made landfall. The medicane then dissipated on September 12th over northwestern Egypt after causing massive destruction from floods in northern Libya and Greece. A record 16” (414 mm) of rainfall was measured in 24 hours at the city of Al-Bayda, Libya, with satellite-derived measurements suggesting locally higher totals near Tolmeita and Battah. Medicane Daniel’s impact resulted in it being the deadliest medicane in history, and the costliest tropical cyclone on record outside of the North Atlantic Ocean. Over $21.14 billion (2023 USD) in damages was reported from Daniel, and over 4,361 fatalities have been confirmed with unofficial estimates of 20,000, and over 7,000 were injured. 
+  </Prose>
+
+  <Figure>
+    <Image
+      src={new URL('./darnah-daniel-satellite.jpg', import.meta.url).href}
+      alt='Medicane Daniel VIIRS imagery, 9 September 2023.'
+      align='left'
+    />
+    <Caption>
+    Satellite imagery of Daniel over the Mediterranean. (VIIRS imagery from NASA Worldview)
+    </Caption>
+
+  </Figure>
+
+</Block>
+
+
+<ScrollytellingBlock>
+  <Chapter
+    center={[17.29046905780905,38.5666921324378]}
+    zoom={6}
+    datasetId='darnah-flood'
+    layerId='darnah-gpm'
+    datetime='2023-09-01'

```

Comment:
This datetime does not match the datetime in the paragraph (the paragraph says September 4th) + As far as I can check from the dataset explorer https://deploy-preview-372--visex.netlify.app/data-catalog/darnah-flood/explore?projection=mercator%7C%7C&basemapid=satellite&datetime=2023-09-07T00%3A00%3A00.000Z The oldest data that can be retrieved from our endpoint is September 7th. What is the intended visual for this chapter (and below)?